src/Livewire/Concerns/RemembersQueries.php

```diff
@@ -15,9 +15,9 @@ trait RemembersQueries
      *
      * @return array{0: mixed, 1: float, 2: string}
      */
-    public function remember(callable $query, string $key = ''): array
+    public function remember(callable $query, string $key = '', int $ttl = 5): array
```

Comment:
Would be nice if this supported all the types that `Cahce::remember` supports for the TTL, not just `int`